.github/workflows/lint.yml

```
@@ -13,11 +13,11 @@ jobs:
         with:
           capture: "junit.xml"
           args: "-t --formatter JUnit"
-          extra_libs: mysql+polyzone+qblocales
+          extra_libs: mysql
```

```suggestion
          extra_libs: mysql+qblocales
```
Linter should know about locales.